doc/contributing.rst

```diff
@@ -1,221 +1,760 @@
-:orphan:
+.. contents:: Page contents
+   :local:
+   :depth: 2
 
-.. include:: links.inc
+.. _contributing:
 
-.. _contribute_to_mne:
+.. include:: ../CONTRIBUTING.rst
```

Comment:
This ends up doing something weird where CONTRIBUTING.rst links to `dev/contributing.html` ... which is exactly what this RST document is (see second paragraph [here](https://12938-1301584-gh.circle-artifacts.com/0/dev/contributing.html)).

I like keeping things DRY, but the lesser of two evils here I think is actually to duplicate content to avoid this self-linking behavior due to the `.. include::` statement including a link to this document. And then in `CONTRIBUTING.rst` and here we can have `..` comments saying modifications to one should be made in the other to keep them consistent.